code/game/objects/items/food/sandwichtoast.dm

```diff
@@ -241,3 +241,34 @@
 	foodtypes = GRAIN | MEAT | DAIRY | VEGETABLES
 	food_flags = FOOD_FINGER_FOOD
 	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/death_sandwich
+	name = "Death sandwich"
+	desc = "Eat it right, or you die!"
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "death_sandwich"
+	food_reagents = list(
+		/datum/reagent/consumable/nutriment = 8,
+		/datum/reagent/consumable/nutriment/protein = 14,
+		/datum/reagent/consumable/nutriment/vitamin = 6,
+	)
+	tastes = list("bread" = 1, "meat" = 1, "tomato sauce" = 1, "death" = 1)
+	foodtypes = GRAIN | MEAT
+	food_flags = FOOD_FINGER_FOOD
+	w_class = WEIGHT_CLASS_SMALL
+	eat_time = 4 SECONDS // I THINK this is longer than normal? Makes it harder to force-feed this to people as a weapon, as funny as that is.
```

Comment:
i think having an `eat_time` like this is an _okay_ safeguard, but we should really do the arnold pizza thing where forcefeeding it doesn't work.

Reply:
arnold pizza has that? I'll have to see how it does that when I wake up tomorrow.

Reply:
last i remember: yes

Reply:
i dont like that idea, if you eat it impropery you get the pain. itd be stupid if you could just be forcefed it

Reply:
I thought you meant that Arnold pizza simply doesn't let you forcefeed other people, the fact that it just bypasses the check and that's it makes me want to keep it the way I did it instead.